src/test/java/io/github/christophermanahan/carnitas/HTTPResponseTest.java

```diff
@@ -0,0 +1,17 @@
+package io.github.christophermanahan.carnitas;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class HTTPResponseTest {
+
+  @Test
+  void createsGETResponseBytes() {
+    Response httpResponse = new HTTPResponse("data");
+
+    byte[] httpResponseBytes = httpResponse.bytes();
+
+    String response = String.format("%s %s %s", HTTPResponse.VERSION, HTTPResponse.GET, HTTPResponse.CRLF);
+    Assertions.assertArrayEquals(response.getBytes(), httpResponseBytes);
```

Comment:
I might be inclined to inline `httpResponseBytes`. Just a suggestion though. Up to you.

```suggestion
    Assertions.assertArrayEquals(response.getBytes(), new HTTPResponse("data").bytes());
```

Reply:
I wanted to setup the arrange act assert explicitly but I cleaned up the naming in the test.